lib/Elastica/Query/MatchQuery.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Elastica\Query;
+
+/**
+ * Match query.
+ *
+ * This class is for forward compatibility reasons. For PHP 8 and above use MatchQuery as Match is reserved.
+ */
+class MatchQuery extends Match
+{
```

Comment:
Well, this is embarrassing. And apparently CI didn't pick this up.

This isn't valid PHP, there's a missing }

Reply:
Yes CI didn't pick it on 6.x as it was broken when Travis stopped testing when quota was reached.
On 7.x CI was migrated to github action.